cypress/e2e/Labels/tests/multipleLabelsCRUDTests.ts

```diff
@@ -0,0 +1,179 @@
+import { checkNoErrorOnThePage } from "../../../support/alertUtils";
+import { buttonClick } from "../../../support/clickUtils";
+import { generateUniqueName } from "../../../support/formUtils";
+import { checkOnCorrectPage } from "../../../support/pageAssertionUtils";
+import { setup } from "../../../support/setupUtils";
+import {
+  BUTTON_TEXT,
+  HEADER_TEXT,
+  QUERY_TEXT_1,
+  QUERY_TEXT_2,
+  UNGROUPED,
+} from "../utilsAndConstants/labelTestConstants";
+import {
+  fillInNewLabelForm,
+  addNewLabelToGroup,
+  labelDelete,
+  checkLabelExists,
+} from "../utilsAndConstants/labelsUtils";
+
+export const MultipleLabelsCRUDTests = () =>
+  describe("Multiple label Creation/Deletion tests", () => {
+    before(() => {
+      setup();
+    });
+
+    describe("Able to Create / Delete an ungrouped label with multiple labels", () => {
+      const label_1 = generateUniqueName("firstLabel");
+      const label_2 = generateUniqueName("secondLabel");
+      const labelList = [label_1, label_2];
+
+      afterEach(() => {
+        checkOnCorrectPage({
+          headerText: HEADER_TEXT.LABELS,
+          notRightPageText: [
+            HEADER_TEXT.CREATE_LABEL,
+            HEADER_TEXT.UPDATE_LABEL,
+          ],
+          notRightPageButton: BUTTON_TEXT.CANCEL,
+        });
+        checkNoErrorOnThePage();
+
+        for (const label of labelList) {
+          labelDelete({
+            groupName: UNGROUPED,
+            labelName: label,
+          });
+          checkLabelExists({
+            labelName: label,
+            groupName: UNGROUPED,
+            doesExist: false,
+          });
+        }
+      });
+
+      it("Create multiple ungrouped labels", () => {
```

Comment:
This test does not clean labels after creation. Is it ok? Am I missing some other place where it is done ?

Reply:
Ah, it does clean them in the afterEach, which runs through and deletes all the labels (line 43)